.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can you remove this file? Everyone's going to have different editor preferences.